File: plop-templates/adapter/acquiring/src/client/model/BaseResponse.java
package com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.client.model;

import lombok.Getter;
import lombok.Setter;

{{#if_eq doc true}}
/**
 * Общие свойства для всех ответов находятся здесь. Наследуй этот класс для выделение специфичных свойств ответов.
 */
{{/if_eq}}
@Getter
@Setter
public class BaseResponse {

}

File: plop-templates/adapter/payout/src/client/model/BaseResponse.kt
package com.rbkmoney.{{adapterPayoutPackageCase bank_name}}.client.model

{{#if_eq doc true}}
/**
 * Общие свойства для всех ответов находятся здесь. Наследуй этот класс для выделение специфичных свойств ответов.
 */
{{/if_eq}}
abstract class BaseResponse(
    val errorCode: Int? = null,
    val errorMessage: String? = null
)
